src/utils/text.ts

```diff
@@ -60,3 +60,39 @@ export function getMatches(regex: RegExp, input: string, group = 0) {
 export function getStringSize(value: string) {
     return value.length * 2;
 }
+
+
+export function formatCSS(text: string) {
+
+    function getShift(depth: number) {
+        if (depth === 0) {
+            return '';
+        }
+        return ' '.repeat(4 * depth);
+    }
+
+    const CSS = (text
+        .replace(/((.*?)\{(\s+)\})|((.*?)\{\})/g, '') // Removing Empty CSS Rules
+        .replace(/\s\s+/g, ' ') // Replacing multiple spaces to one
+        .replace(/\{/g,'{%--%') // {
+        .replace(/\}/g,'%--%}%--%') // }
+        .replace(/\;(?![^\(]*\)|[^\"]*\")/g,';%--%') // ; and do not target between () mostly for url()
+        .replace(/\,(?![^\(]*\)|[^\"]*\")/g,',%--%')
+        .replace(/%--%\s{0,}%--%/g,'%--%') // Remove %--% Without any characters between it to the next %--%
+        .split('%--%'));
```

Comment:
Think if it is possible to use `\n` instead of `%--%`

Reply:
Already formatted text e.g. if you want have it on devtools CSS section it can mess up that's why the special selected characters